src/main/java/org/jitsi/jicofo/Participant.java

```diff
@@ -112,7 +113,7 @@
      * returns <tt>false</tt>.
      */
     private Map<String, IceUdpTransportPacketExtension> transportMap
-        = new HashMap<>();
+        = new ConcurrentHashMap<>();
```

Comment:
What is it that the ConcurrentHashMap solves exactly? I see a check on a get which decides whether to perform a put and my concern in mostly whether get and put will be protected as a whole block, not whether they are individually protected. Anyway, I need more clarity on the goal here.